replit_river/server_transport.py

```diff
@@ -148,30 +146,69 @@ async def _establish_handshake(
                 websocket,
             )
             raise InvalidMessageException("handshake request to wrong server")
-        if handshake_request.expectedSessionState is None:
-            # TODO: remove once we have upgraded all clients
-            my_session_id = await self._get_or_create_session_id(
-                request_message.from_, handshake_request.sessionId
-            )
-        elif handshake_request.expectedSessionState.reconnect:
-            maybe_my_session_id = await self._get_existing_session_id(
-                request_message.from_,
-                handshake_request.sessionId,
-                handshake_request.expectedSessionState.nextExpectedSeq,
+
+        oldSession = self._sessions.get(request_message.from_, None)
```

Comment:
```suggestion
        old_session = self._sessions.get(request_message.from_, None)
```

python likes snake_case for locals. same everywhere a new local is created.